packages/web-components/src/tab/tab.styles.ts

```diff
@@ -57,6 +58,11 @@ export const styles = css`
     font-weight: ${fontWeightSemibold};
   }
 
+  :host::before,
+  :host::after {
+    height: ${strokeWidthThicker};
+  }
```

Comment:
There are no styles applied to `::before`, is there a reason we need to apply it that? Similarly, I'm curious if this can be added below given that's when there's actually an `:after` element.

Reply:
I decided to move it into two instances of `:host([aria-selected='true/false']:hover)::after` as they definitely need dimensions.